meson.build

```diff
@@ -48,7 +48,7 @@ conf_data.set('INSTALL_PREFIX', get_option('prefix'))
 conf_data.set('PLUGIN_PATH', join_paths(get_option('prefix'), get_option('libdir'), 'wayfire'))
 metadata_dir_suffix = 'share/wayfire/metadata'
 conf_data.set('PLUGIN_XML_DIR', join_paths(get_option('prefix'), metadata_dir_suffix))
-conf_data.set('SYSCONFDIR', join_paths('/', get_option('sysconfdir')))
+conf_data.set('SYSCONFDIR', join_paths(get_option('prefix'), get_option('sysconfdir')))
```

Comment:
I think the intent is that --prefix can be set independently of --sysconfdir. For example, with your proposal, you wouldn't be able to set --prefix to /usr and SYSCONFDIR to /etc. With the current state, it shouldn't be a problem if you wanted to set sysconfdir relative to prefix, because prefix is known at configure time (e.g. --prefix=/usr --sysconfdir=/usr/etc). That said, is there a specific case where this might not work?

Reply:
`--prefix=/usr` defaults to `/etc` and `--sysconfdir=/usr/etc` works fine. I suspect `join_paths()` is magical. ;)